src/content/getusermedia/resolution/js/main.js

```diff
@@ -52,6 +59,16 @@ var fullHdConstraints = {
 function gotStream(mediaStream) {
   window.stream = mediaStream; // stream available to console
   video.srcObject = mediaStream;
+  messagebox.style.display = 'none';
+  videoblock.style.display = 'block';
+}
+
+function errorMessage(who, what) {
+  let message = who + ': ' + what;
+  messagebox.innerHTML = message;
```

Comment:
i'd go for .innerText here -- mostly because .innerHTML is 👿

Reply:
done